### ISSUE TITLE
Subscribing to Webhook Events

### ISSUE THREAD

**Issue by JamesAdamHughes** [hozn/stravalib#109]
One of the most useful features of the Strava v3 API are the webhook events: http://strava.github.io/api/v3/events/ 
These allow an app to subscribe to events from the user, such as uploading an activity. This means the server does not have to poll strava constantly to check if a new activity has been uploaded.

Currently this is not in the python stravalib, but would be a great addition to an already great library! Are there any plans to integrate this?

**Comment by JamesAdamHughes** [comment]
I've just found this already has been done in client.py ln:1382! Awesome

**Comment by hozn** [comment]
Yes, definitely the docs need some love. Let us know if you find any issues. This does seem like a great feature (someone contributed this code, so I haven't used it myself)

**Comment by hozn** [comment]
@JamesAdamHughes , do you have any example code you'd like to share that's working for you?  -- I'm interested in updating the docs to include this.

**Comment by hozn** [comment]
The docs need to be improved, but clearing the milestone.  I'm going to be using this in coming weeks, so I'll take that opportunity to document it.

**Comment by yass-arafat** [comment]
Is there any documentation added regarding subscribing event? Thanks @hozn

**Comment by hawkeye217** [comment]
+1 for even a quick snippet of code on subscribing to webhook events!

**Comment by hozn** [comment]
Thanks all for your patience!  I don't have documentation per se, but happy to link over to an example of where I used this for a project: https://github.com/freezingsaddles/freezing-nq/blob/master/freezing/nq/api/webhook.py

**Comment by rangan2510** [comment]
Would really like a flask example for webhooks.
